pandas/src/parser/tokenizer.c

```diff
@@ -1387,13 +1355,7 @@ int tokenize_whitespace(parser_t *self, size_t line_limit)
 
         case START_RECORD:
             // start of record
-            if (skip_this_line(self, self->file_lines)) {
-                self->state = SKIP_LINE;
-                if (c == '\n') {
-                    END_LINE();
-                }
-                break;
-            } else  if (c == '\n') {
+            if (c == '\n') {
```

Comment:
@jreback : I'm going to leave this one alone for a separate PR, as I think there is a larger issue to be dealt with in terms of `delim_whitespace` and `lineterminator` per my comments in #12912.